phpunit/html/wp-html-walker-test.php

```diff
@@ -0,0 +1,1012 @@
+<?php
+/**
+ * Unit tests covering WP_HTML_Walker functionality.
+ *
+ * @package WordPress
+ * @subpackage HTML
+ */
+
+if ( ! function_exists( 'esc_attr' ) ) {
+	function esc_attr( $s ) {
+		return str_replace( '"', '&quot;', $s );
+	}
+}
+
+if ( ! class_exists( 'WP_UnitTestCase' ) ) {
+	abstract class WP_UnitTestCase extends \PHPUnit\Framework\TestCase {}
+}
+
+require_once __DIR__ . '/../../lib/experimental/html/index.php';
+
+/**
+ * @group html
```

Comment:
```suggestion
 * @group html
 *
 * @coversDefaultClass WP_HTML_Walker
```
Note: The use of `@coversDefaultClass` means you don't need to prefix the method with the classname for method level `covers` annotations. See [PHPUnit Annotations - @coversDefaultClass](https://phpunit.readthedocs.io/en/9.5/annotations.html#coversdefaultclass)

Can we get `covers` annotations on each test method for accurate coverage reports to get this ready for merge? See [Trac 56070](https://core.trac.wordpress.org/ticket/56070) for the order/spacing of annotations.

Reply:
I just added the `covers` annotations in 81d43e26fd